ui/app/pages/home/index.scss

```
@@ -27,12 +27,12 @@
   }
 
   &__main-view &__tab {
+    @include H7;
```

Did you mean to use `H6` here? That would be closer to the old size. It looks strangely small at 12px.

```suggestion
    @include H6;
```